### DISCUSSION TITLE
Performance measurements

### DISCUSSION THREAD

**Discussion by AlexGusevEpam** [pmndrs/valtio#730] [General]
Hello
I tried to use this library for the first time
My use case was to manage frequently changing Map with 10^4 values and I realized almost all delays were connected to `proxyMap` function
So I think it would be helpful for others to get some performance analysis depending on complexity to help people to choose between state management libraries.

**Comment by dai-shi** [comment]
Yeah, it would be interesting. Hope someone can try that.

fwiw, https://github.com/krausest/js-framework-benchmark shows some comparison.
